src/components/Login/Login.tsx

```diff
@@ -37,7 +37,16 @@ const Login = ({ displayAsButtons = false }) => {
     }
 
     try {
-      await connectWallet();
+      const connectWalletHandler = new Promise(resolve => {
+        resolve(connectWallet());
+      });
+
+      toast.promise(connectWalletHandler, {
+        pending: "Logging in",
+        success: "Logged in ✅",
```

Comment:
I would remove these icons with the success and error messages because the toast notification should already have an icon in the top left corner.